src/snovault/schema_utils.py

```diff
@@ -389,7 +389,8 @@ def combine_schemas(a, b):
         if a[name] == b[name]:
             combined[name] = a[name]
         elif name == 'type':
-            combined[name] = sorted(set(ensurelist(a[name]) + ensurelist(b[name])))
+            combined[name] = sorted(
```

Comment:
I know this is autoformatting. I would probably format it like: 
```
combined[name] = sorted(
    set(ensurelist(a[name]) + ensurelist(b[name]))
)
```
Not a big deal.

Reply:
I agree, looks better! Just changed